packages/data-service-generator/src/server/resource/create-dtos.ts

```diff
@@ -54,17 +54,18 @@ export async function createDTOModulesInternal({
     Object.values(entityDTOs)
   );
 
-  for (const dto of entityDTOs) {
-    const isEnumDTO = namedTypes.TSEnumDeclaration.check(dto);
-    let module: Module;
-    if (isEnumDTO) {
-      module = createEnumDTOModule(dto, dtoNameToPath);
-    } else {
-      module = createDTOModule(dto, dtoNameToPath);
-    }
+  console.time(`createDTOModulesInternal-ALL ${dtos.length}`);
+  await Promise.all(
+    entityDTOs.map((dto) => {
+      const isEnumDTO = namedTypes.TSEnumDeclaration.check(dto);
+      return isEnumDTO
+        ? createEnumDTOModule(dto, dtoNameToPath)
+        : createDTOModule(dto, dtoNameToPath);
+    })
+  ).then((modulesRes) => modulesRes.forEach((module) => moduleMap.set(module)));
```

Comment:
Do we need an `await` on the `moduleMap.set()` call? (I am asking since the original code awaited this)